### PR TITLE
added .env support, replaced config, added dotenv typings, added dotenv.

### LINKED ISSUE #12: [Proposal] - Change ecosystem.config file to use .env
**Is your feature request related to a problem? Please describe.**
Instead of using ecosystem.config files we should consider moving to .env instead. We may need this for Heroku or other things anyway so we should consider making the change.

**Describe the solution you'd like**
Add .env files to hold the config information.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by WallyThePenguin**
## Description
I removed configs from imports, added .env example, index.ts needs a bit of cleaning due to  it. added typings, added dotenv to packages
Closes #12 
## List of Changes

# Screenshots of Changes

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1 @@
+token='token_here'
```
Comment (Skillz4Killz) [inline .env.example]: Double check if u even need quotes here please

File: index.ts [hunk 2]
```diff
@@ -1,4 +1,20 @@
...
+  case 'test':
+    path = `${__dirname}/../../.env.test`;
+    break;
+  case 'production':
+    path = `${__dirname}/../../.env.production`;
+    break;
+  default:
+    path = `${__dirname}/../../.env.development`;
+}
+dotenv.config({ path });
```
Comment (Skillz4Killz) [inline index.ts]: Why 2 configs? 1 above empty then another here?

File: index.ts [hunk 3]
```diff
@@ -1,4 +1,20 @@
...
+export const token = process.env.token;
```
Comment (Skillz4Killz) [inline index.ts]: Why is this exported?

Comment (WallyThePenguin) [reply index.ts]: This is if we we're exporting this as a constant, we don't need this, but if we export all of the environment variables, we can just import and use them in other files like a password, and bot support author id's

File: index.ts (hunk 2 above)
Comment (WallyThePenguin) [reply index.ts]: `dotenv.config();` is basically setting up all the cases, `dotenv.config({ path });` is parsing the path's in the cases, if I don't set this config up, it doesn't allow process.env.token

File: index.ts (hunk 3 above)
Comment (WallyThePenguin) [reply index.ts]: also, if you dislike using process.env a lot, this loses the process and use them as a const, token.

**Pull request merged**